src/bracketing.jl

```diff
@@ -262,7 +259,7 @@ function find_zero(fs, x0, method::Bisection;
                    p=nothing,
                    tracks = NullTracks(),
                    verbose=false,
-                   kwargs...) where {M <: Union{Bisection}}
+                   kwargs...)
```

Comment:
I guess at some point the function signature was `method::M, ...` and it was forgotten to remove the where clause when it was changed.

Reply:
Hmm, this one may be hiding something. There is a different criteria for exact bisection. Let me look.

Reply:
Yup, you guessed right